installers/olm/config/examples/pgadmin.example.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PGAdmin
+metadata:
+  name: example-pgadmin
+  namespace: openshift-operators
+spec:
+  dataVolumeClaimSpec:
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  serverGroups:
+    - name: "Crunchy Postgres for Kubernetes"
```

Comment:
Would a comment about "This will grab all postgresclusters" be helpful?

Reply:
Might be a good idea. I thought about it a bit, but we don't have anything similar in our other examples. I can try a test and see how it looks.

Reply:
To follow up, I did a test and our current bundle generation method strips out comments, so I don't know that there's a simple way to add that at the moment.

Reply:
+1 with a quick look I saw nothing that jumped out at me - is there something that you want me to focus on?

Reply:
Thanks for taking a look. Mainly the line in `installers/olm/description.md`. Is that sufficient or is there anything else you think is worth mentioning there?